src/WebKit_Performance_001.test.js

```diff
@@ -0,0 +1,88 @@
+import {
+  pluginDeactivate,
+  pluginActivate,
+  webKitBrowserActions,
+  settingUrl,
+} from './commonMethods/commonFunctions'
+import {
+  validateFPS,
+  initializeSamplesArray,
+  fetchFPS,
+  calcAvgFPS,
+} from './commonMethods/webKitPerformanceCommonFunctions'
+import constants from './commonMethods/constants'
+
+const URL = 'http://webkit.org/blog-files/3d-transforms/poster-circle.html'
+let minFPS = 40
+
+export default {
+  setup() {
+    initializeSamplesArray()
+  },
+  title: 'WPEWebkit performance poster circle',
+  description: 'Loads the Poster Circle CSS3 animation and measures its performance',
+  steps: [
+    {
+      description: 'Deactivating monitor',
+      test: pluginDeactivate,
+      params: constants.monitorPlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Deactivating WebKitBrowser',
+      test: pluginDeactivate,
+      params: constants.webKitBrowserPlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Activating WebKitBrowser',
+      test: pluginActivate,
+      params: constants.webKitBrowserPlugin,
+      assert: 'suspended',
+    },
+    {
+      description: 'Resuming WebKitBrowser',
+      test: webKitBrowserActions,
+      params: constants.resume,
+      assert: 'resumed',
+    },
```

Comment:
Consider putting steps for preparing the test environment (that don't really test anything) in a single setup method (same goes for teardown steps).

Possibly use the this.$sequence-helper (example: https://github.com/WebPlatformForEmbedded/Storm-Testcases/blob/master/src/helpers/goToAppstore.js)

Reply:
Moved this code to the setup() method by using the sequence helper.